module-apps/apps-common/widgets/ProgressTimerWithBarGraphAndCounter.cpp

```diff
@@ -18,11 +18,31 @@ namespace app
     {
         updateText();
         updateProgress();
+        updateTimeWidget();
         ProgressTimer::update();
     }
 
     void ProgressTimerWithBarGraphAndCounter::updateText()
     {
+        using utils::time::Duration;
+        if (text == nullptr) {
+            return;
+        }
+        const auto secondsRemaining = duration - std::chrono::duration_cast<std::chrono::seconds>(elapsed);
```

Comment:
As both are `std::chrono`'s durations, why not do something like:
```Cpp
    const auto secondsRemaining = std::chrono::duration_cast<std::chrono::seconds>(duration - elapsed);

Reply:
The code was restored form inital commit introducing(deleting this code) TimeFixedWidget so I did not want to change anything but its ok change I will change to:   
   const auto secondsRemaining = std::chrono::duration_cast<std::chrono::seconds>(duration - elapsed);
  
as you suggested